packages/fluentui/docs/src/examples/components/Toolbar/Visual/ToolbarExampleVariables.shorthand.tsx

```diff
@@ -1,10 +1,18 @@
-import { Toolbar } from '@fluentui/react-northstar';
+import { Toolbar, toolbarMenuBehavior, Accessibility, ToolbarMenuBehaviorProps } from '@fluentui/react-northstar';
 import * as React from 'react';
 import { CallVideoIcon, MoreIcon, SkypeLogoIcon } from '@fluentui/react-icons-northstar';
 
+// behavior is overridden, as focus was causing inconsistency in displaying focus outline for screener tests
```

Comment:
is this intentionally in the Tree PR?

Reply:
Don't think so... So I am removing... if I am wrong @kolaps33  please revert the commit